pkg/controller/plan/builder/vsphere/builder.go

```diff
@@ -45,8 +45,8 @@ func (r *Builder) Secret(vmID string, in, object *core.Secret) (err error) {
 	if hostFound {
 		hostURL := liburl.URL{
 			Scheme: "https",
-			Host: host.Spec.IpAddress,
-			Path: vim25.Path,
+			Host:   host.Spec.IpAddress,
```

Comment:
Has anything really changed on these lines?

Reply:
No.  The makefile runs `gofmt` and looks like these lines changed in an earlier PR that got merged without a final `gofmt`.